anta/tests/logging.py

```diff
@@ -190,8 +190,15 @@ class VerifyLoggingLogsGeneration(AntaTest):
     description = "Verifies if logs are generated."
     categories: ClassVar[list[str]] = ["logging"]
     commands: ClassVar[list[AntaCommand | AntaTemplate]] = [
-        AntaCommand(command="send log level informational message ANTA VerifyLoggingLogsGeneration validation", ofmt="text"),
-        AntaCommand(command="show logging informational last 30 seconds | grep ANTA", ofmt="text", use_cache=False),
+        AntaCommand(
+            command="send log level informational message ANTA VerifyLoggingLogsGeneration validation",
+            ofmt="text",
+        ),
+        AntaCommand(
+            command="show logging informational last 30 seconds | grep ANTA",
+            ofmt="text",
+            use_cache=False,
+        ),
```

Comment:
This happens in most of your PRs :P Your ruff settings or versions might be different than the repo maybe?